packages/react-performance/src/test/PerformanceReport.test.tsx

```diff
@@ -36,7 +36,7 @@ describe('<PerformanceReport />', () => {
     timer.runAllTimers();
 
     const [fetchedUrl, {body, method, headers}] = fetch.lastCall();
-    expect(fetchedUrl).toBe(url);
+    expect(fetchedUrl).toBe(`${url}/`);
```

Comment:
Is there a rationale for updating these assertions to these more strict patterns in the context of this PR?

Reply:
Since we are doing strict comparison, the new fetch mock returned the forward slash. The tests failed without it. Perhaps they could be changed to `toContain`?

Reply:
I guess it's better to be strict!

Reply:
But then we might need to document the migration path if existing test can break.

Reply:
e.g. https://github.com/Shopify/quilt/blob/a74673d6b1e887d005639beabb29f4369aaff6ea/packages/react-i18n/CHANGELOG.md#changed-2

Reply:
Thanks for your guidance @atesgoral! I added a migration guide. It doesn't have a date on it as this hasn't been released yet!